content/docs/Reference/env-variables.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Environment Variables"
+linkTitle: "Environment Variables"
+date: 2019-03-30
```

Comment:
I don't personally see much value in merging a page like this has no useful information. The tokens page looks useful.

However if we merge the environment variables page, then we should set the page in the future so that our publishing framework (Hugo) doesn't actually make the page visible until it's ready.

For example set the date to: 2025-01-01

Reply:
@sharifsalah You can let me know the useful information needed to be added in the environment variable's page and I will add it accordingly. 
I included the page as it was mentioned in the issue but I don't exactly know the description of each variable required for documentation.